EnglishFilter/sections/specific.txt

```diff
@@ -1748,7 +1748,7 @@ hdporn.org##a[href^="https://www.hdporn.org/link/"]
 boxrec.com##.midAdvert
 ||upsieutoc.com/images/2020/08/19/bl.jpg
 uii.io,wordcounter.icu,genpas.icu,shurt.pw##body > iframe
-redgifs.com,imgbaron.com,picbaron.com##iframe[src^="//tsyndicate.com/"]
+myadultanimes.com,redgifs.com,imgbaron.com,picbaron.com##iframe[src^="//tsyndicate.com/"]
```

Comment:
It seems this rule does nothing

Reply:
I get popups without this rule.

Reply:
There is a blank leftover with AdGuard Apps without this rule.

Reply:
Ok, I see